### PR TITLE
vendor: Bumped CoverBee to v0.3.0 and cilium/ebpf to v0.10.0

### PR DESCRIPTION
**Pull request by dylandreimerink**
Updated CoverBee to v0.3.0, this version is dependant on cilum/ebpf@v0.10.0 so this PR bumps it as well so everything will still compile.

```release-note
Bumped CoverBee to v0.3.0 and cilium/ebpf to v0.10.0
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -17,10 +17,10 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/ec2 v1.74.0
 	github.com/aws/smithy-go v1.13.4
 	github.com/blang/semver/v4 v4.0.0
-	github.com/cilium/coverbee v0.2.2
+	github.com/cilium/coverbee v0.3.0
 	github.com/cilium/customvet v0.0.0-20221207003726-576d50314db5
 	github.com/cilium/deepequal-gen v0.0.0-20200406125435-ad6a9003139e
-	github.com/cilium/ebpf v0.9.4-0.20221102092914-a9cf21df64c2
+	github.com/cilium/ebpf v0.10.0
```
Comment (rolinh) [inline go.mod:23]: Given `cilium/ebpf` is also bumped, I think it should appear in the PR description/release notes.

Comment (dylandreimerink) [reply go.mod:23]: Agreed, see my comment here: https://github.com/cilium/cilium/pull/23203#issuecomment-1400221958

**Review by ti-mo — APPROVED** [review]
Ack given it's rebased after #23203 is merged.

File: go.mod (hunk 1 above)
Comment (dylandreimerink) [reply go.mod:23]: Nevermind, rephrased the commit message and PR description to reflect the change in both libs

**Review by rolinh — APPROVED** [review]

**Comment by ldelossa** [comment]
An very elusive "full suite pass". Impressive.

**Pull request merged**